README.md

```diff
@@ -7,11 +7,26 @@ The installer also bundles a full Julia version manager called `juliaup`. One ca
 ## Using Juliaup
 
 If you want to try it, here is what you should do:
-- Make sure you don't have any version of Julia on your PATH. `Juliaup` will handle all `PATH` related aspects of your Julia installation.
+
+### Windows Users
+Make sure you don't have any version of Julia on your PATH. `Juliaup` will handle all `PATH` related aspects of your Julia installation.
+
+#### Option 1: The Microsoft Store
 - Install Julia from the Windows Store [here](https://www.microsoft.com/store/apps/9NJNWW8PVKMN).
 
 Once you have that installed, `julia` is on the `PATH`, there is a start menu shortcut and it will show up as a profile in Windows Terminal. Any of those will start Julia. The VS Code extension will also automatically find this Julia installation.
 
+#### Option 2: User Level Installation Script
+- Run `powershell -nop -c iex (New-Object System.Net.WebClient).DownloadString('https://raw.github.com/Julialang/juliaup/master/juliaup-init.sh.cmd')` 
+    or [download and run](https://raw.github.com/Julialang/juliaup/master/juliaup-init.sh.cmd) to install juliaup.
+- Note, it's preferable to choose `yes` to _add juliaup.exe and julia.exe to user PATH_. This will allow the VS Code extension and other programs to access `julia`.
+
+### Linux and macOS Users
+- Run `curl https://raw.github.com/Julialang/juliaup/master/juliaup-init.sh.cmd | sh` or [download and run](https://raw.githubusercontent.com/Julialang/juliaup/master/juliaup-init.sh.cmd) to install juliaup.
+- Optionally, add the `~/.juliaup/bin` binaries to your `PATH`. Preferably, add symlinks via sudo privileges: `sudo ln "$HOME/.juliaup/bin/juliaup" /usr/local/bin/juliaup -s; sudo ln "$HOME/.juliaup/bin/julialauncher" /usr/local/bin/julia -s`.
```

Comment:
Good point, it's messy symlinking user-level things to system-wide directories. Will think a bit about this... It might be worth asking the user if the path should be appended in ~/.bashrc like conda and rust does is. I have a strange feeling about installing it flat into `$HOME/.local/bin/`, since I want to be able to delete the whole directory (such as `~/.juliaup/bin`) and not have side effects.